media/css/sandstone/sandstone-resp.less

```diff
@@ -166,7 +166,48 @@ a.more {
             }
         }
     }
+}
+
+// mozID visual style
+// full width rows w/differing bg colors
+// TODO: needs responsive styles
```

Comment:
Nope! This page isn't responsive, and isn't using this file. However, I thought it would be a good to at least start the styles here for when we need it on a responsive page.

Should I remove for now, or leave in for the next person to continue?